src/UnlockEligibility.sol

```diff
@@ -2,253 +2,256 @@
 pragma solidity ^0.8.19;
 
 // import { console2 } from "forge-std/Test.sol"; // comment out before deploy
-import { HatsEligibilityModule, HatsModule, IHatsEligibility } from "../lib/hats-module/src/HatsEligibilityModule.sol";
-import { IPublicLock } from "../lib/unlock/smart-contracts/contracts/interfaces/IPublicLock.sol";
-import { IUnlock } from "../lib/unlock/smart-contracts/contracts/interfaces/IUnlock.sol";
-import { ILockKeyPurchaseHook } from "../lib/unlock/smart-contracts/contracts/interfaces/hooks/ILockKeyPurchaseHook.sol";
-import { ILockKeyTransferHook } from "../lib/unlock/smart-contracts/contracts/interfaces/hooks/ILockKeyTransferHook.sol";
-
-contract UnlockEligibility is HatsEligibilityModule, ILockKeyPurchaseHook, ILockKeyTransferHook {
-  /*//////////////////////////////////////////////////////////////
+import {HatsEligibilityModule, HatsModule, IHatsEligibility} from "../lib/hats-module/src/HatsEligibilityModule.sol";
```

Comment:
Dang. My linter does not seem to match yours :/

Reply:
you can run `forge fmt` to lint according to the `[fmt]` config in foundry.toml